src/components/TableModule/TableHeaderCell.tsx

```diff
@@ -80,6 +80,13 @@ export const useStyles = makeStyles(
     rotatedIcon: {
       transform: 'rotate(180deg)',
     },
+    isSticky: {
+      background: theme.palette.graphite[50],
+      position: 'sticky',
+      left: 0,
+      zIndex: 4,
```

Comment:
how'd you arrive at this? Magic z-index numbers can be problematic. One way to at least help with this is to create a constants file with them, so they're easy to reference around a project, and exported for consumers.

Reply:
I'm going to fix this in another PR, so I can also move the Modal and Select Z-indexes